src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopyArray = [];
+  let stateCopy = Object.assign({}, state);
+
+  for (const i of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Please provide descriptive names